src/effects/How to make an effect stateless.txt

```diff
@@ -21,7 +21,7 @@ Let use use EffectEcho as an example.
    In cpp file:
        Insert sufficient auto& echoSettings = mSettings; (which changes later) and echoSettings. (which won't need more change) to fix compilation
         
-       Remove call to Reset() in constructor
+       Remove call to Parameters().Reset(*this); in constructor
```

Comment:
What makes it correct at this step to remove this call -- is that the initializations it formerly did now happen in the constructor of `EffectEchoSettings` because you use in-struct initializers with the same default values as in the `EffectParmeters`.

So that is worth mention too in the guide.